vaadin-grid-flow-parent/vaadin-grid-testbench/src/main/java/com/vaadin/flow/component/grid/testbench/GridTRElement.java

```diff
@@ -31,14 +35,35 @@ public class GridTRElement extends TestBenchElement {
      */
     public GridTHTDElement getCell(GridColumnElement column) {
         TestBenchElement e = (TestBenchElement) executeScript(
-                "const grid = arguments[0];" //
+                "const row = arguments[0];" //
                         + "const columnId = arguments[1];" //
-                        + "return Array.from(grid.children)."
+                        + "return Array.from(row.children)."
                         + "filter(function(cell) { return cell._column && cell._column.__generatedTbId == columnId;})[0]",
                 this, column.get__generatedId());
         return e == null ? null : e.wrap(GridTHTDElement.class);
     }
 
+    /**
+     * Gets the cells for the given columns in this row.
+     *
+     * @param columns
+     *            the column elements
+     * @return a {@link GridTHTDElement} list with the cells for the given
+     *         columns
+     */
+    public List<GridTHTDElement> getCells(GridColumnElement... columns) {
```

Comment:
nit: Almost the whole method screams warning

<img width="1016" alt="Screenshot 2023-10-10 at 13 58 42" src="https://github.com/vaadin/flow-components/assets/1222264/c3aedf96-7d52-442b-802d-a56404338ff9">

Could store the result of `executeScript` as an `Object` and then cast that separately to avoid this.